docs/deployment.rst

```diff
@@ -53,6 +53,10 @@ Restrictions
 Borg is instructed to restrict clients into their own paths:
 ``borg serve --restrict-to-path /home/backup/repos/<client fqdn>``
 
+The client will be able to access any file or subdirectory inside of ``/home/backup/repos/<client fqdn>``
+but no other directories.  You can allow a client to access several directories by passing multiple
+`--restrict-to-path` flags, for instance: ``borg serve --restrict-to-path /home/backup/repos/<client fqdn>/root --restrict-to-path /home/backup/repos/<client fqdn>/home``
+
```

Comment:
deployment.rst is just an example - should this be rather in the reference docs?

also, maybe mention that one can just create several repo subdirectories in there, so the client has multiple repos but just one --restrict-to-path.